integration_tests/steps/device.py

```diff
@@ -95,7 +95,7 @@ def step_impl(context):
     # placing bids and offers on every market cycle.
     # Should stop if an error occurs or if the simulation has finished
     counter = 0  # Wait for five minutes at most
-    while context.device.errors == 0 and context.device.status != "finished" and counter < 900:
+    while context.device.errors == 0 and context.device.status != "finished" and counter < 300:
```

Comment:
I don't think this is enough for the integration tests to finish running, but feel free to skip it for now , I'll test it locally once possible

Reply:
This is only for one integrationtest. 5 Minutes should be plenty actually. Will not be needed in 3398 neither.